components/common/ButtonEmitWithDropdown.vue

```diff
@@ -80,6 +85,23 @@ export default {
       )
         this.toggleOpen();
     },
+    handleiOSBlur(e) {
+      if (
+        [
+          'iPad Simulator',
```

Comment:
- [ ] We should create an enum and a map for this in case we need it at some other places as well